server/views/pages/explore.njk

```diff
@@ -3,7 +3,16 @@
 {% block body %}
 <div class="row">
   <div class="container">
-    <h1>Explore</h1>
+    <h1>From the Blog</h1>
+    <ul>
+      {% for post in wpPosts %}
+        <li>
+          <a href="/articles/wp/{{ post.slug }}">{{ post.title }}</a>
+        </li>
+      {% endfor %}
+    </ul>
+
+    <h1>From the Collection</h1>
```

Comment:
I guess this won't end up as a prod page, but having two `h1`s like this makes my palms clammy. I guess theoretically you could put each inside e.g. a `section`. Not sure if that'd make my situation any better, though!